cypress/integration/tests/developer/applicationinventory/assessment/filter.test.ts

```diff
@@ -251,4 +252,31 @@ describe("Application inventory filter validations", { tags: "@tier2" }, functio
         exists(application1.name);
         clickByText(button, clearAllFilters);
     });
+
+    it("Artifact type filter validations", function () {
+        // For application must have Binary group,artifact and version
+        const application2 = new Application(
+            getRandomApplicationData("tackleTestApp_Source", {
+                binaryData: this.appData[2],
+            })
+        );
+        application2.create();
+        cy.get("@getApplication");
+        cy.wait(2000);
+
+        // Check application exists on the page
+        exists(application2.name);
+
+        // Apply artifact filter check with associated artifact field
+        applySearchFilter(artifact, "Associated artifact");
+        cy.wait(2000);
+        exists(application2.name);
+        clickByText(button, clearAllFilters);
```

Comment:
Also check for an application that does not have associated artifact .
doesNotExist(applicationwithoutartifact);

Reply:
Done.. Added below for Not  associated artifact " notExists(application.name)"